charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 11.1.1
+version: 11.1.2
```

Comment:
Wouldn't this be `11.2.0` ?

rest everything looks good to me. but will wait for one more reviewer as well.